modules/ROOT/partials/misc/advcode-shortcuts.adoc

```diff
@@ -1,13 +1,22 @@
-== Advanced Code Editor search and replace keyboard shortcuts
+== Enhanced Code Editor search and replace keyboard shortcuts
 
-Advanced Code Editor provides the following shortcuts within the code dialog.
+Enhanced Code Editor provides the following shortcuts within the code dialog.
 
 [cols=",,",options="header"]
 |===
-|Action |PC |Mac
-|Find |Ctrl+F |Command+F
-|Find next instance |Ctrl+G |Command+G
-|Find previous instance |Shift+Ctrl+G |Shift+Command+G
-|Replace |Ctrl+H |Command+Option+F
-|Replace All |Shift+Ctrl+R |Shift+Command+Option+F
+|Action |PC |Mac 
+|Find |Ctrl+F |Command+F 
+|Find next instance |Ctrl+G |Command+G 
+|Find previous instance |Shift+Ctrl+G |Shift+Command+G 
+|Replace |Ctrl+H |Command+Option+F 
+|Replace All |Shift+Ctrl+R |Shift+Command+Option+F 
+|Open Search Panel |Ctrl+F |Command+F 
+|Find Next |F3 |F3 
+|Find Previous |Shift+F3 |Shift+F3 
+|Close Search Panel |Esc |Esc 
+|Select Selection Matches |Ctrl+Shift+L |Command+Shift+L 
+|Select Next Occurrence |Ctrl+D |Command+D 
+|Select Next Occurrence (Atom) |Ctrl+B |Command+B 
+|Fold Code |Ctrl+Q |Command+Q+Fn
+|Accept Completion |Tab |Tab 
```

Comment:
@lorenzo-pomili Can you provide an updated list of what we are going to include in this shortcut list based on the recent changes?